deployment/terraform_azure/variables.tf

```diff
@@ -28,3 +28,11 @@ variable "enable_auto_scaling" {
   default = true
   type    = bool
 }
+
+//sql
+variable "sql_admin_user" {
+  default = "sql_admin"
+}
+
+variable "sql_admin_password" {
```

Comment:
Could you make it generate a random DB password - that's then output.

Reply:
updated